buildah_intro/buildah_intro.sh

```diff
@@ -249,7 +297,7 @@ buildah_from_dockerfile() {
     read -p "buildah run hello-working-container python3 /home/HelloFromContainer.py"
     buildah run hello-working-container python3 /home/HelloFromContainer.py
 
-    echo 
+    echo
     echo "Now a quick advertisement from Podman."
     echo "Let's run the container using Podman."
     echo
```

Comment:
The `podman run hello` here in the *_rootless failed with:

```
Now a quick advertisement from Podman.
Let's run the container using Podman.

podman run hello
error creating libpod runtime: exit status 125
```

The other (original) one runs fine.

Reply:
Strange. it works for me.